src/dynarec/rv64/dynarec_rv64_660f.c

```diff
@@ -2463,13 +2463,7 @@ uintptr_t dynarec64_660F(dynarec_rv64_t* dyn, uintptr_t addr, uintptr_t ip, int
                 ed = x1;
                 wback = x3;
             }
-            if (rv64_zbs)
-                BEXT_(x4, ed, gd);
```

Comment:
Sorry, but the BEXT macro cannot be used because this is the "66" version of the opcode, so operating on 16bits register. The non-rex.w mask is 0xf and not 0x1f here.

Reply:
Oh, I'm sorry.... then:

```
            if (rv64_zbs) {
                if (rex.w) {
                    BEXT_(x4, ed, gd);
                } else {
                    ANDI(x2, gd, 0xf);
                    BEXT_(x4, ed, x2);
                }
            } else {
                ANDI(x2, gd, rex.w ? 0x3f : 0xf);
                SRL(x4, ed, x2);
                ANDI(x4, x4, 1);
            }
```

Reply:
Fixed in the newest code. Thanks!